index.html

```diff
@@ -6782,490 +6780,4 @@ <h4 style="margin-top: 10px;">Posters</h4>
                                      </tr>
                                      <tr class="abstract">
                                          <td colspan="3">
-                                             <p>Dynamic Time Warping (<strong>DTW</strong>, [Sakoe et al. 1978]) is a distance measure for two time series of different lengths that allows non-linear alignments of time series and has been applied in many fields (text mining, activity recognition, transport research, etc.) for detection of motifs, classification and clustering. In contrast to the Euclidean distance, the complexity of DTW computation is quadratic in the number of time points. There are many approaches on speeding up the calculation of DTW (early abandoning, approximation, lower bound). Many applications generate streaming data, e.g. transport mode detection. For streaming data, an efficient incremental computation of DTW is desirable, which our R package <strong>IncDTW</strong> (Incremental Dynamic Time Warping) facilitates by recycling already computed results. We used the Rcpp package to implement the heart of the DTW algorithm in C++. We collected smartphone accelerometer data while travelling with different transport modes and demonstrate that DTW outperforms the traditional Euclidean distance in detecting transport mode specific patterns, and how the computation time benefits from the incremental calculations. In terms of computation time we compare our implementation with existing ones in R and demonstrate our method to be up to 20 times faster.</p>
```

Comment:
Seems like this PR deleted the rest of the HTML file. Can you pls fix it?

Reply:
yes, sorry. I'm fixing it now.